virt/virtual_machines/virt-create-vms.adoc

```diff
@@ -31,6 +31,9 @@ Refer to the virtual machine wizard fields section when running the web console
 
 :virtualmachine:
 include::modules/virt-vm-wizard-fields-web.adoc[leveloffset=+2]
+
+Enable the xref:../../scalability_and_performance/using-cpu-manager.adoc#using-cpu-manager[CPU Manager] to use the high-performance workload profile.
```

Comment:
I am thinking it would not be obvious to someone scanning the document that this text is related to a table entry.

Because formatting this text as a [footnote under the table](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#alternative-footnote-styling-in-tables) is not an option because of the xref, I would suggest adding the xref to the Additional resources section and rewording the table entry to say something like this: High-Performance (requires CPU Manager; see the Additional resources section for more information).